index.html

```diff
@@ -1,42 +1,157 @@
+                    <li class="list-item"><img src="images/angelist.png" alt="follow me on angellist" width="20" height="20" ></li>
+                </ul>
+                </div>
+        </section>  
+
+            <section class="work-section">
+               <div class="card">
+            <div class="section-img"><img src="images/snapshoot1.png"></div>
+            <div class="card-info">
+            <h2 class="work-header">Tonic</h2>
+            <ul class="descr-list">
+                <li class="descr-list-item1">CANOPY</li>
+                <li class="descr-list-item"><img class="counter-img" src="images/Counter.png"> End Dev</li>
+                <li class="descr-list-item"><img class="counter-img"src="images/Counter.png">2015</li>
+            </ul>
+            <p class="lang-descr">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="lang-list">
+                <li class="lang-list-descr">html</li>
+                <li  class="lang-list-descr">css</li>
+                <li  class="lang-list-descr">javaScript</li>
+            </ul>
+            <button class="btn-project">See Project</button>
+        </div>
+        </div>
+
+        <div class="card">
+            <div class="section-img"><img src="images/snapshoot2.png"></div>
+            <div class="card-info">
+            <h2 class="work-header">Multi-Post
+                Stories</h2>
+            <ul class="descr-list">
+                <li class="descr-list-item1">CANOPY</li>
+                <li class="descr-list-item"><img class="counter-img" src="images/Counter.png"> End Dev</li>
+                <li class="descr-list-item"><img class="counter-img"src="images/Counter.png">2015</li>
+            </ul>
+            <p class="lang-descr">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="lang-list">
+                <li class="lang-list-descr">html</li>
+                <li  class="lang-list-descr">css</li>
+                <li  class="lang-list-descr">javaScript</li>
+            </ul>
+            <button class="btn-project">See Project</button>
+        </div>
+        </div>
+        <div class="card">
+            <div class="section-img"><img src="images/snapshoot3.png"></div>
+            <div class="card-info">
+            <h2 class="work-header">Tonic</h2>
+            <ul class="descr-list">
+                <li class="descr-list-item1">CANOPY</li>
+                <li class="descr-list-item"><img class="counter-img" src="images/Counter.png"> End Dev</li>
+                <li class="descr-list-item"><img class="counter-img"src="images/Counter.png">2015</li>
+            </ul>
+            <p class="lang-descr">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="lang-list">
+                <li class="lang-list-descr">html</li>
+                <li  class="lang-list-descr">css</li>
```

Comment:
- [ Optional ] Good Job implementing the change pointed by the previous reviewer. however, I suggest you add a margin in between your cards, to stick to the Figma design. You'll agree with me that the lack of spacing does not give a good appeal from a user perspective, and we can fix that by added the margin provided on the Template. Find the difference from the image attached. Well done 👏 

## Figma Design 
![image](https://user-images.githubusercontent.com/50990925/212218392-415bb83c-82aa-477a-8596-9f4e9393d8cd.png)

## You present Design
![image](https://user-images.githubusercontent.com/50990925/212218561-2942ddc0-3742-4c6f-b273-3b6e8fb61599.png)

- [ ] I strongly suggest you make the changes for all the cards in your project to give it a nice user view. Good work 👏